q2_longitudinal/plugin_setup.py

```diff
@@ -48,7 +56,19 @@
     'state_1': Str,
     'state_2': Str,
     'parametric': Bool,
-    'replicate_handling': Str % Choices(['error', 'random', 'drop']),
+    **{k: v for k, v in miscellaneous_parameters.items()
+       if k == 'replicate_handling'},
+}
+
+miscellaneous_parameter_descriptions = {
+    'state_column': ('Metadata column containing state (e.g., Time) '
+                     'across which samples are paired.'),
```

Comment:
Yes, thanks for checking — do you think this needs to be clarified further here?

Reply:
The word "paired" threw me off because I initially thought it meant that there were only two states (e.g. "before" and "after"). I see now that these samples are being "paired" sequentially. I don't have a suggestion to make it clearer, fine to leave as-is unless my thought process above helps prompt some text changes :)